src/messages.rs

```diff
@@ -33,34 +34,44 @@ pub struct WebDriverError {
     pub stacktrace: Option<String>,
 }
 
+#[derive(Serialize, Default)]
+pub struct Capabilities {
+    alwaysMatch: BTreeMap<String, JsonValue>,
+}
+
 #[derive(Serialize)]
 pub struct NewSessionCmd {
-    capabilities: JsonValue,
+    capabilities: Capabilities,
 }
 
 impl NewSessionCmd {
     pub fn new() -> Self {
```

Comment:
Take a Capabilities value as an argument here, I think.

Reply:
If the concern is breaking changes then we can also remove new() and add it at a later time, and just leave default.